src/geometry.cpp

```diff
@@ -463,7 +465,7 @@ BoundaryInfo distance_to_boundary(Particle& p)
 extern "C" int openmc_find_cell(
   const double* xyz, int32_t* index, int32_t* instance)
 {
-  Particle p;
+  Geometron p;
```

Comment:
Might be good to rename the variable to better reflect the fact it is no longer a full particle (`geom_state` instead of `p`?)

Reply:
Hmm yes good point!